hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/CreateOperationSuite.java

```diff
@@ -340,12 +337,8 @@ HapiSpec childContractStorageWorks() {
                     final var retResults = function.decodeReturn(resultBytes);
                     String contractIDString = null;
                     if (retResults != null && retResults.size() > 0) {
-                        final var retValHex = ((Address) retResults.get(0)).toString();
-                        final var retVal = Bytes.fromHexString(retValHex).toArray();
-
-                        final var realm = Longs.fromByteArray(Arrays.copyOfRange(retVal, 4, 12));
-                        final var accountNum = Longs.fromByteArray(Arrays.copyOfRange(retVal, 12, 20));
-                        contractIDString = String.format("%d.%d.%d", realm, 0, accountNum);
+                        contractIDString =
+                                ((Address) retResults.get(0)).toString().substring(2);
```

Comment:
Well, previous lines L343-348 were certainly nasty to see _right in the middle of a test!!_  **Glad** they're gone.  (Here's that `.substring(2)` again though ...)